src/Gtk/Avalonia.Gtk/GtkPlatform.cs

```diff
@@ -115,7 +115,7 @@ public IPopupImpl CreatePopup()
 
         public IRenderer CreateRenderer(IRenderRoot root, IRenderLoop renderLoop)
```

Comment:
I'm not sure that `CreateRenderer` should live in platform code. I'd rather move it to `ITopLevelImpl` instead.